koin/src/main/java/in/koreatech/koin/ui/navigation/KoinNavigationDrawerActivity.kt

```diff
@@ -497,6 +471,15 @@ abstract class KoinNavigationDrawerActivity : ActivityBase(),
         startActivity(intent)
     }
 
+    private fun goToLoginActivity() {
+        val intent = Intent(
+            this,
+            LoginActivity::class.java
+        )
+        finishAffinity()
+        startActivity(intent)
```

Comment:
```suggestion
      Intent(this, LoginActivity::class.java).apply {
          finishAffinity()
          startActivity(this)
      }
```
축약해서 사용하면 더 좋을 것 같네용

Reply:
감사합니다!